Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift

```diff
@@ -37,7 +37,10 @@ public enum PaymentMethodType: RawRepresentable, Hashable, Codable {
     case econtextStores
     case econtextATM
     case econtextOnline
-    case boleto
+    case boletoBancario
```

Comment:
Technically, this is a breaking change.
Maybe keep the old one as-is, just mark as depricated?

Reply:
As discussed, we keep `boleto` but mark it as deprecated